LoggingKata/TacoParser.cs

```diff
@@ -6,13 +6,37 @@
     public class TacoParser
     {
         readonly ILog logger = new TacoLogger();
-        
+
         public ITrackable Parse(string line)
         {
-            logger.LogInfo("Begin parsing");
+            var cells = line.Split(',');
+            if (cells.Length < 3 || cells.Length > 3)
+            {
+                return null;
+            }
+                logger.LogInfo("Begin parsing");
+
+            TacoBell newTacoBell = new TacoBell();
+            {
+                double Latitude = double.Parse(cells[0]);
+                double Longitude = double.Parse(cells[1]);
```

Comment:
If the ShouldFailParse method were written, these lines wouldn't be passing the tests